src/adr/articles/0015-multi-region-default.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: layouts/adr.njk
+title: "Multi Region by Default"
+weight: 1
+date: 2022-04-08
+review_in: 12 months
+tags:  
+    - adr
+    - data_privacy
+    - security_threats
+    - techniques
+    - deployment
+    - stability
+areas_of_coverage: ["Digital Service"]
+status: "accepted"
+contributors: ["John Nolan"]
+adr_number: "0015"
+---
+
+## Context
+
+We want our services to be resilient to data center failure. If a data center or region goes down, we want to be able to switch to a new region quickly with minimal to no data loss.
+
+Region failure is rare but does happen. Common situations for region failure are
+
+* Natural disaster
+* DDOS attacks
+* Data Center bugs
+
+### Decision
+
+Our other services already are or are in the process of using Multi Region Architecture.
+
```

Comment:
QQ - can we add links to this ADR later once we have written one for those live services?

Reply:
Yeah definitely, any existing decisions I am trying to sign post so can do a PR after linking to other team decisions